docs/assets/js/main.js

```diff
@@ -75,9 +75,45 @@ function injectFooterCopywrite() {
     footer.innerHTML = `&copy;2008-${new Date().getFullYear()} Expensify, Inc.`;
 }
 
+// Count property of --scroll-y variable
+window.addEventListener('scroll', () => {
+    document.documentElement.style.setProperty('--scroll-y', `${window.scrollY}px`);
+});
+
+function openSidebar() {
+    document.getElementById('sidebar-layer').style.display = 'block';
+    document.getElementById('gsc-i-id1').placeholder = 'Search for something...';
+    // Make body unscrollable
```

Comment:
Add a space between a comment an a block of code
```suggestion

    // Make body unscrollable
```